frontend/src/components/button/Button.tsx

```diff
@@ -40,10 +55,10 @@ export default function Button(props: Props) {
 					setIsLoading(false);
 				} }>
 			{ props.icon && !props.placeIconRight && <i className={ isLoading ? "fi fi-rr-spinner spin" : props.icon }
-                                                        style={ { marginRight: "var(--spacing)" } }/> }
+                                                        style={ { ...{ marginLeft: "calc(var(--spacing) * -0.75)" }, ...props.iconStyle } }/> }
```

Comment:
why this magical number?

Reply:
Because it has to be a fraction of the usual spacing.